projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/_overrides.scss

```diff
@@ -139,3 +138,44 @@
 .admin-color-classic-bright.folded #adminmenu .toplevel_page_site-notices .wp-menu-image {
 	background-color: $studio-gray-80 !important;
 }
+
+/**
+ * Styles for All Sites menu
+ */
+#wp-admin-bar-my-sites {
```

Comment:
Is there any reason why we have to duplicate the styles here?

Reply:
It is no longer needed, so I removed it: https://github.com/Automattic/jetpack/pull/36632#discussion_r1558792590. 

📝 Initially, as detailed in the PR description, the duplication was needed because `currentColor` wasn't utilized. c.f., https://github.com/Automattic/jetpack/pull/36632#discussion_r1557015871